firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -74,13 +77,7 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        navigationItem.title = .Settings.Search.Title
-
-        // To allow re-ordering the list of search engines at all times.
-        tableView.isEditing = true
-        // So that we push the default search engine controller on selection.
-        tableView.allowsSelectionDuringEditing = true
+        subscribeToRedux()
```

Comment:
This needs to be behind a feature flag. The old code should continue as it is today if the feature flag is disabled.